src/core_modules/capture-core/reducers/descriptions/form.reducerDescription.js

```diff
@@ -205,26 +205,24 @@ export const formsSectionsFieldsUIDesc = createReducerDescription({
     [newPageActionTypes.CLEAN_UP_DATA_ENTRY]: cleanUp,
     [rulesEffectsActionTypes.UPDATE_RULES_EFFECTS]: (state, action) => {
         const { formId, rulesEffects } = action.payload;
-        const formSectionFields = state[formId];
```

Comment:
I would assume `formSectionFields` was used here for a reason; what makes you think it can be removed?

Reply:
I shouldn’t have removed it, but now it's back in place. Thanks for the feedback!